### PR TITLE
Elio

### PR DESCRIPTION
**Pull request by elio303**

### CODE REVIEW

File: a2.css [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
 	text-align: center;
 	width: 30%;
 }
+
+
+/******************************************************
+ GAME PAGE
+ ******************************************************/
+ 
+ div.infoBar{
```
Comment (wwjzxwjc) [inline a2.css]: is there an extra space here

Comment (elio303) [inline a2.css]: Yeah

File: a2.html [hunk 2]
```diff
@@ -1,35 +1,44 @@
...
+			<form>
+				<input type="submit" name="Start Game" value="Start Game"></input>
+			</form>
+			<p id="highScore">High Score: </p>
+		</body>
+		<div id="infoBar">
+                	<div id="timer">
+               			<!-- Some JS function called here so that a timer can be used -->
+                	</div>
+                	<button type="button">Pause</button>
```
Comment (wwjzxwjc) [inline a2.html]: we could use an icon here, when we click it, we will just change the href of the img icon to play icon.

Comment (elio303) [inline a2.html]: Yeah that works. Let's keep it a button for now and then change it with JS onclick.

Comment (elio303) [inline a2.html]: I like your icon idea. Do you want to make it an icon that says pause and then when onclick -function to JS - change the icon to resume?

Comment (wwjzxwjc) [inline a2.html]: i think it will just a toggle function. fix the empty space thing, then merge with master.

Comment (elio303) [inline a2.html]: What's the empty space thing? So a JS toggle function? Should I make it?

**Pull request merged**